instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -188,13 +188,19 @@ def get_or_create_headers():
 
```

Comment:
Probably want to also remove `upper()` call in `get_default_span_name()` to be consistent.